engine/time/src/main/java/io/deephaven/time/DateTimeUtils.java

```diff
@@ -1741,7 +1742,9 @@ public static LocalDate convertDate(String s) {
     }
 
     /**
-     * Converts a DateTime String from a few specific zoned formats to a DateTime
+     * Converts a {@link DateTime} String from a few specific zoned formats to a {@link DateTime}.
```

Comment:
@jmao-denver I think the associated pydocs need to get updated to indicate that the new format is also supported.